UXO_Kismet_SF_3.tex

```diff
@@ -0,0 +1,363 @@
+    \item \control{}: Engage the key switch and power on the control boxes.
+    \item \control{}: Open the Tank Vent Valve.
+    \item \control{}: Open the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If leaks are observed:
+        \begin{checklist}
+            \item \control{}: Close the Remote Fill Valve.
+            \item \primary{}: Open the Line Vent Valve using the ropes.
+            \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+            \item \primary{} and \secondary{}: Approach the test plumbing.
+            \item \primary{}: Close the Cylinder Valve.
+            \item \control{}: Open the Remote Fill Valve.
+            \item \daq{}: Confirm that the Fill Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit plumbing setup.
+        \end{checklist}
+        \item If the Remote Fill Valve fails to open:
+        \begin{checklist}
+            \item \ops{}: Abort test procedures and revisit control system setup.
+        \end{checklist}
+    \end{checklist}
+    \item \secondary{}: Proceed only when a white plume is visible from the Tank Vent Valve.
+    \item \control{}: Close the Tank Vent Valve.
+    \item \control{}: Close the Remote Fill Valve.
+    \begin{checklist}[label=$\bullet$]
+        \item If the Remote Fill Valve fails to close:
+        \begin{checklist}
+            \item \primary{} and \secondary{}: Approach the test plumbing.
+            \item \primary{}: Close the Series Fill Valve.
+            \item \primary{} and \secondary{}: Retreat to the test control area, behind the blast shield.
+        \end{checklist}
+    \end{checklist}
+    \item \heat{}: Open the Tank Heating Valve.
+    \item \daq{}: Proceed only when the oxidizer tank pressure is at least 800 psi.
+    \begin{checklist}[label=$\bullet$]
+        \item If the oxidizer tank pressure does not reach 800 psi:
+        \begin{checklist}
+            \item \heat{}: Close the Tank Heating Valve.
+            \item \primary{}: Open the Line Vent Valve using the ropes.
+            \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+            \item \primary{} and \secondary{}: Approach the test plumbing.
+            \item \primary{}: Close the Cylinder Valve.
+            \item \control{}: Open the Tank Vent Valve.
+            \item \control{}: Open the Remote Fill Valve.
+            \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+            \item \ops{}: Abort test procedures and revisit water jacket setup.
+        \end{checklist}
+    \end{checklist}
+    \item \heat{}: Close the Tank Heating Valve.
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \control{}: Perform engine startup procedure:
+    \begin{checklist}
+        \item Arm the Primary Ignition switch.
+        \item Hold down the Fire button until black smoke is observed.
```

Comment:
Also get him to read out current reading here - that bit _should_ still work

Reply:
oak